src/util/typeAssertions.ts

```diff
@@ -46,4 +44,4 @@ export const isPath = (fabricObject?: FabricObject): fabricObject is Path => {
 export const isActiveSelection = (
   fabricObject?: FabricObject
 ): fabricObject is ActiveSelection =>
-  !!fabricObject && Object.hasOwn(fabricObject, 'multiSelectionStacking');
+  !!fabricObject && 'multiSelectionStacking' in fabricObject;
```

Comment:
Yes i was commenting while you posted this, and indeed it also make more sense for the subclasses.
While 'multiSelectionStacking' or 'source' are a property that get defined over and over through the getDefaults() in the contructors, that may not be the case if getDefaults() is emptied and you decide to work on the prototype chain with values. In that case on a subclass ownProperty would fail, but in no.

Reply:
Yeah I used `in` as well because for the `filler.source` case it should be an own property typically, whereas with the ActiveSelection case it's even safer if the class is subclassed